spec/models/grant_spec.rb

```diff
@@ -2,16 +2,13 @@
 
 RSpec.describe Grant, type: :model do
   describe "Grant.list" do
-    let(:status) { GrantStatus.create id: 1, description: "Incomplete" }
-    let(:agency1) { Agency.create }
-    let(:agency2) { Agency.create }
-    let(:admin) { User.create role: "admin" }
-    let(:worker1) { User.create email: "test1@test.com", password: "testtest", agency: agency1 }
-    let(:worker2) { User.create email: "test2@test.com", password: "testtest", agency: agency1 }
-    let(:worker3) { User.create email: "test3@test.com", password: "testtest", agency: agency2 }
-    let(:grant1) { Grant.create total_rent: 1000.0, status: status, user_id: worker1.id }
-    let(:grant2) { Grant.create total_rent: 1000.0, status: status, user_id: worker2.id }
-    let(:grant3) { Grant.create total_rent: 1000.0, status: status, user_id: worker3.id }
+    let(:admin) { create(:user, :admin) }
+    let(:worker1) { create(:user, :case_worker) }
+    let(:worker2) { create(:user, :case_worker, agency: worker1.agency) }
+    let(:worker3) { create(:user, :case_worker) }
+    let(:grant1) { create(:grant, user: worker1) }
+    let(:grant2) { create(:grant, user: worker2) }
+    let(:grant3) { create(:grant, user: worker3) }
     before do
```

Comment:
I created a factory called agency_with_grants, number_of grants.  This could be used hear to avoid the setup in the test.  I can also refactor when my stuff gets put in.